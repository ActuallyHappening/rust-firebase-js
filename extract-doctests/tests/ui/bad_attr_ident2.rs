use extract_doctests::extract_doctests;

#[extract_doctests(inline_config(bad_attr = ""))]
fn placeholder() {}

fn main() {}
