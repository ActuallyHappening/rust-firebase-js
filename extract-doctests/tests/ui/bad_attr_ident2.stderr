error: Failed to parse attributes as Config; #[extract_doctest] checks for inline configuration declerations before reading your Cargo.toml file from disk, but the inline configuration was not valid. The exact error is attached below.

 --> tests/ui/bad_attr_ident2.rs:3:20
  |
3 | #[extract_doctests(inline_config(bad_attr = ""))]
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: Expected template, not bad_attr
 --> tests/ui/bad_attr_ident2.rs:3:34
  |
3 | #[extract_doctests(inline_config(bad_attr = ""))]
  |                                  ^^^^^^^^
